test/schemes/boundary/open_boundary/boundary_zone.jl

```diff
@@ -0,0 +1,221 @@
+@testset verbose=true "Boundary Zone" begin
+    @testset verbose=true "Boundary Zone 2D" begin
+        particle_spacing = 0.2
+        open_boundary_layers = 4
+
+        plane_points_1 = [[0.0, 0.0], [0.5, -0.5], [1.0, 0.5]]
+        plane_points_2 = [[0.0, 1.0], [0.2, 2.0], [2.3, 0.5]]
+
+        @testset "Points $(i)" for i in eachindex(plane_points_1)
+            point_1 = plane_points_1[i]
+            point_2 = plane_points_2[i]
+
+            plane_size = point_2 - point_1
+
+            flow_directions = [
+                normalize([-plane_size[2], plane_size[1]]),
+                -normalize([-plane_size[2], plane_size[1]]),
+            ]
+
+            @testset "Flow Direction $(j)" for j in eachindex(flow_directions)
+                inflow = InFlow(; plane=(point_1, point_2), particle_spacing,
+                                flow_direction=flow_directions[j], density=1.0,
+                                open_boundary_layers)
+                outflow = OutFlow(; plane=(point_1, point_2), particle_spacing,
+                                  flow_direction=flow_directions[j], density=1.0,
+                                  open_boundary_layers)
+
+                boundary_zones = [
+                    inflow,
+                    outflow,
+                ]
+
+                @testset "$boundary_zone" for boundary_zone in boundary_zones
+                    zone_width = open_boundary_layers *
+                                 boundary_zone.initial_condition.particle_spacing
+                    sign_ = (boundary_zone isa InFlow) ? -1 : 1
+
+                    @test plane_points_1[i] == boundary_zone.zone_origin
+                    @test plane_points_2[i] - boundary_zone.zone_origin ==
+                          boundary_zone.spanning_set[2]
+                    @test sign_ * flow_directions[j] ≈
+                          normalize(boundary_zone.spanning_set[1])
+                    @test zone_width ≈ norm(boundary_zone.spanning_set[1])
```

Comment:
```suggestion
                    @test isapprox(zone_width, norm(boundary_zone.spanning_set[1]), atol=1e-14)
```
Same above and for the other errors. Better set an explicit tolerance.